README.md

```diff
@@ -70,13 +70,6 @@ object(s)`. Then, insert it with:
 {% include mdl_img.html %}
 ```
 
-Update the list of packages of the Docker image `scalingo-18`:
```

Comment:
Why did you remove it, it's no longer relevant?

Reply:
As stated in my commit message, it has been moved to Notion, in the procedure to release a new stack version. I don't think it's really relevant here